wire-ios/Wire-iOS/Sources/Authentication/Coordinator/AuthenticationCoordinator.swift

```diff
@@ -501,18 +501,9 @@ extension AuthenticationCoordinator {
 
     // MARK: - Modals
 
-    /// Opens the browser and reopens the current alert upon dismissal if needed.
+    /// Opens the URL in the selected browser
     private func openURL(_ url: URL) {
```

Comment:
is it only password reset that uses this function?

Reply:
As far as I can tell, only the password reset uses the `AuthenticationCoordinatorAction.openURL(_:)` action.

![Screenshot 2024-10-28 at 12 07 01](https://github.com/user-attachments/assets/1c80ab86-b5d1-42cc-a48c-c621f90ce434)

Reply:
As far as I can tell, only the password reset uses the `AuthenticationCoordinatorAction.openURL(_:)` action, and only the `AuthenticationCoordinatorAction.executeActions(_:)` method calls it.
![Screenshot 2024-10-28 at 12 07 01](https://github.com/user-attachments/assets/7d492db9-3dc5-4265-a7d9-0545052eefaa)